app-backend/db/src/main/scala/com/azavea/rf/database/ExportDao.scala

```diff
@@ -98,9 +98,9 @@ object ExportDao extends Dao[Export] {
       )
     }
 
-    val exportInput = (export.projectId, export.toolRunId) match {
-      // Exporting a tool-run
-      case (_, Some(toolRunId)) => Right(astInput(toolRunId, user))
+    val exportInput = (export.projectId, export.analysisId) match {
+      // Exporting an analysis
+      case (_, Some(analysisId)) => Right(astInput(analysisId, user))
```

Comment:
This isn't really an idiomatic use of `Either` -- normally, `Right`s are successes and `Left`s are failures, while here, `Right`s and `Left`s are both successes. Also, could you add a type annotation to `exportInput` to make it more clear what's going on in the for comprehension below?

Reply:
Don't really want to refactor that part ATM, since I'm just trying to get the new stuff working.
I may address this later if I have time.

Reply:
That's fine for getting things working, but this does need to be changed -- `Left`s are for error conditions, and using it to hold non-errors will confuse people and make the `Either` functor weird

Reply:
I'll make an issue.

Reply:
I'm throwing up my hands given that this seems to be the strategy until [union types](http://dotty.epfl.ch/docs/reference/union-types.html) are available. I don't like it still but I've been convinced.